src/pytest_inmanta_lsm/lsm_project.py

```diff
@@ -398,6 +409,13 @@ def lsm_global_cache_reset(self, original_global_cache_reset_method: typing.Call
         # Monkeypatch the client because it was just re-created by the reset function
         self.monkeypatch_client()
 
+    def get_entity_version(self, service_entity: str, version: typing.Optional[int] = None) -> int:
+        """
+        Returns the version to use for this service entity
+        """
+
+        return version if version is not None else self.service_entities_default_versions.get(service_entity, 0)
+
```

Comment:
How about just adding an entry `(service_entity, None)` in the `service_entities` dict, then there is no need for this manual step and the `service_entities_default_versions` dict?

Reply:
Changed `self.service_entities` to have `(name, None)` stored as the default version. This duplicates the service entity in the dict, but I feel like it is easier to manage